STAT697-01_s20-team-2_data_preparation.sas

```diff
@@ -175,39 +174,33 @@ https://github.com/yxie18-stat697/team-2_project_repo/blob/master/data/enr16.xls
 %loadDatasets
 
 
-/* check gradaf17 for bad unique id values, where the columns COUNTY, DISTRICT, 
-and SCHOOL form a composite key */
+/* Check gradaf17 for bad unique id values, where the columns 'COUNTY', 
+'DISTRICT', and 'SCHOOL' form a composite key. After executing this query, we 
+see that gradaf17_raw_dups has no missing unique id component */
 proc sql;
-    /* check gradaf17 for duplicate unique id values; after executing this 
-    query, we see that gradaf17_raw_dups has no missing unique id component */
-    create table gradaf17_raw_dups as
+create table gradaf17_raw_dups as
         select
-            COUNTY
+             COUNTY
             ,DISTRICT
             ,SCHOOL
-            ,count(*) as row_count_for_unique_id_value
-        from
-            gradaf17
+            ,count(*) 
+            as row_count_for_unique_id_value
```

Comment:
Yes, this variable is the correct format. snake_case

Reply:
Thanks for the feedback. It's very helpful for me to visualize what you meant.